docs/_docs/refs/mojito-locales.md

```diff
@@ -32,9 +32,12 @@ permalink: /docs/refs/mojito-locales/
 | bn-IN	        | Bengali (India) |
 | bs-BA	        | Bosnian (Bosnia and Herzegovina) |
 | ca-ES	        | Catalan (Spain) |
+| chr-021	        | Cherokee (Northern America) |
+| cr-021	        | Cree (Northern America) |
```

Comment:
My current thinking is to not have region/territory unless it is a locale that is available in CLDR. I know currently Mojito always have a region but I'm thinking to move away from that pattern.

The new locale list would be any locale defined in: https://github.com/unicode-cldr/cldr-core/blob/master/availableLocales.json + any languages: https://github.com/unicode-cldr/cldr-localenames-full/blob/master/main/en/languages.json. 

So for the few i checked they'd be only languages

thoughts?

Reply:
I think that's fair. I'd love to support someone on getting the most appropriate language codes into CLDR itself through formal registration process, rather than doing something opinionated here :)